From 9c9b78361df1c53ced433eb1ff074e14fe44577a Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Sun, 16 Jan 2022 08:15:36 +0000
Subject: [PATCH 90/94] WIP: dt-bindings: arm: amlogic: add support for MeCool
 M8S Pro W

The MeCool M8S Pro W is an Android STB based on the Amlogic P281
(GXL) reference design with an S905W chip.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index 65732fdd7e47..937ab7ab1cdc 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -89,6 +89,7 @@ properties:
               - amlogic,p281
               - oranth,tx3-mini
               - jethome,jethub-j80
+              - videostrong,gxl-m8s-pro-w
           - const: amlogic,s905w
           - const: amlogic,meson-gxl
 
-- 
2.35.1

